### DISCUSSION TITLE
Twitter feeds that use twitter's widgets.js to render the content

### DISCUSSION THREAD

**Discussion by molecula2788** [FreshRSS/FreshRSS#6142] [General]
There are some solutions for turning a twitter list into an rss feed (like rss.app). However, looking at the resulting feed, each item makes use of the `https://platform.twitter.com/widgets.js` script that will do the actual work of rendering the tweet. Is there a way to make it work in FreshRSS? By default it will strip all the script tags.

**Comment by Alkarex** [comment]
Maybe look at https://github.com/RSS-Bridge/rss-bridge/blob/master/docs/10_Bridge_Specific/TwitterV2.md
I do not know whether it still works, though

**Reply by molecula2788** [reply]
Thanks, I'll take a look at it.

Regarding the alternative of using twitter's `widgets.js`, could there be a way to allow it to run? Maybe with an extension? (I'm not familiar yet with the extension model to be able to tell what they can or cannot do). Or if I modify some code myself? I'm running my own self hosted instance, so I would be open to this approach as well.
